content/en/docs/examples/virtual-machines/index.md

```diff
@@ -97,36 +97,34 @@ $ mysql -u root -ppassword test -e  "update ratings set rating=1 where reviewid=
 +----------+--------+
  {{< /text >}}
 
-## Find out the IP address of the VM that will be used to add it to the mesh
+## Expose the mysql service to the mesh
 
-On the VM:
+When the virtual machine is started, it will automatically be registered into the mesh.
+However, just like when creating a Pod, we still need to create a Service before we can easily access it.
 
 {{< text bash >}}
-$ hostname -I
+$ cat <<EOF | kubectl apply -f - -n vm
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysqldb
+  labels:
+    app: mysqldb
+spec:
+  ports:
+  - port: 3306
+    name: tcp
+  selector:
+    app: mysqldb
+EOF
 {{< /text >}}
 
-## Registering the mysql service with the mesh
-
-On a host with access to [`istioctl`](/docs/reference/commands/istioctl) commands, register the VM and mysql db service
-
-{{< text bash >}}
-$ istioctl register -n vm mysqldb <ip-address-of-vm> 3306
-I1108 20:17:54.256699   40419 register.go:43] Registering for service 'mysqldb' ip '10.150.0.5', ports list [{3306 mysql}]
-I1108 20:17:54.256815   40419 register.go:48] 0 labels ([]) and 1 annotations ([alpha.istio.io/kubernetes-serviceaccounts=default])
-W1108 20:17:54.573068   40419 register.go:123] Got 'services "mysqldb" not found' looking up svc 'mysqldb' in namespace 'vm', attempting to create it
-W1108 20:17:54.816122   40419 register.go:138] Got 'endpoints "mysqldb" not found' looking up endpoints for 'mysqldb' in namespace 'vm', attempting to create them
-I1108 20:17:54.886657   40419 register.go:180] No pre existing exact matching ports list found, created new subset {[{10.150.0.5  <nil> nil}] [] [{mysql 3306 }]}
-I1108 20:17:54.959744   40419 register.go:191] Successfully updated mysqldb, now with 1 endpoints
-{{< /text >}}
-
-Note that the 'mysqldb' virtual machine does not need and should not have special Kubernetes privileges.
-
 ## Using the mysql service
 
 The ratings service in Bookinfo will use the DB on the machine. To verify that it works, create version 2 of the ratings service that uses the mysql db on the VM. Then specify route rules that force the review service to use the ratings version 2.
```

Comment:
use the DB on the **virtual** machine

or maybe change to `VM` as we use that in the next sentence.